screens/Settings.js

```diff
@@ -567,7 +567,7 @@ class Settings extends React.Component {
             />
           </ListItem>
 
-          <ListItem bottomDivider
+          {/* <ListItem bottomDivider
```

Comment:
Merging it veerji, but would be cool if we can mention the reason here too in one line.